src/gui/dialogs/setupwizardpages/SetupPageLayout.ts

```diff
@@ -12,9 +12,13 @@ interface SetupPageLayoutAttrs {
 export class SetupPageLayout implements Component<SetupPageLayoutAttrs> {
 	view({ attrs, children }: Vnode<SetupPageLayoutAttrs>): Children {
 		return m("section.center.flex.flex-column.dialog-height-small.mt", [
-			// Sanitizing the images removes the classes from the svg for unknown reasons
-			// So the svg is used raw for now
-			m(".onboarding-logo.center-h", { class: attrs.class }, m.trust(attrs.image)),
+			m("img.onboarding-logo.center-h", {
```

Comment:
do we need `alt` or some indication that it can be ignored by the screen readers?

Reply:
Good catch, it should tell the screen reader to ignore it because it is decorative. [Apparently an empty `alt` attribute will suffice](https://www.w3.org/WAI/tutorials/images/decision-tree/).